src/igvfd/searches/configs/matrix_file.py

```diff
@@ -25,6 +86,12 @@ def matrix_file():
             'lab': {
                 'title': 'Lab'
             },
+            'file_set': {
+                'title': 'File Set'
+            },
+            'content_summary': {
+                'title': 'Content Summary'
+            },
             'status': {
```

Comment:
Should upload_status also be added to columns?

Reply:
upload_status is already in the columns